spec/integration_spec.rb

```diff
@@ -80,6 +87,44 @@
     end
   end
 
+  context "using plain tcp connection" do
+    it "should successfully connect to the server if ssl explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host, :ssl => false)
+      }.not_to raise_error
+    end
+
+    it "should fail to connect to the server if ssl not explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host)
+      }.to raise_error(RuntimeError, /Must set a ssl certificate/)
+    end
```

Comment:
just connecting will be ok (as long as no data is send), as server is waiting for handshake. The time the client sends a message, it will error with unknown response frame (SSL - handshake packet from SSL server). Let's see if I can construct something.